app/src/main/java/com/ahgitdevelopment/course/customexamples/features/screens/AppList.kt

```diff
@@ -2,10 +2,9 @@ package com.ahgitdevelopment.course.customexamples.features.screens
 
 import android.content.Intent
 import androidx.compose.foundation.layout.Column
-import androidx.compose.runtime.Composable
+import androidx.compose.runtime.*
```

Comment:
We should have the same code style configuration because I'm not allowing this kind of imports with `.*`
<img width="938" alt="image" src="https://user-images.githubusercontent.com/5006126/188603233-d0f58b29-1a80-4b19-bbd8-aa4f8923fd7a.png">